bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -87,6 +87,50 @@ public Builder withScan(Scan scan) {
       return this;
     }
 
+    /*
+     * Overrides of the CloudBigtableConfiguration.Builder's with*() so that they return
+     * CloudBigtableScanConfiguration.Builder.
+     */
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Builder withProjectId(String projectId) {
+      return (Builder) super.withProjectId(projectId);
```

Comment:
We ran into a bizarre problem with Generics.  Builder<?> != Builder in some cases; I don't know what kind of JVM quirk caused that, but Builder vs. Builder<?> caused some compilation problems fora  user.   This change fixes that problem.